examples/parse_api/src/app/controllers/home.controller.ts

```diff
@@ -0,0 +1,128 @@
+import {
+    ApiController,
+    Controller,
+    HttpGet,
+    SendsResponse,
```

Comment:
Unused imports

Reply:
Need to use smarter IDE

Reply:
VS Code ESLint plugin should do the trick